plugins/catalog/src/alpha/EntityOverviewPage.tsx

```diff
@@ -19,11 +19,17 @@ import { useEntity } from '@backstage/plugin-catalog-react';
 import Grid from '@material-ui/core/Grid';
 import React, { useMemo } from 'react';
 import { parseFilterExpression } from './filter/parseFilterExpression';
+import { ApiHolder, useApiHolder } from '@backstage/core-plugin-api';
+import useAsync from 'react-use/lib/useAsync';
 
 interface EntityOverviewPageProps {
   cards: Array<{
     element: React.JSX.Element;
-    filterFunction?: (entity: Entity) => boolean;
+    fullWidth?: boolean;
+    filterFunction?: (
+      entity: Entity,
+      context: { apis: ApiHolder },
```

Comment:
There are cases where the Catalog API needs to be called to get more information about the entity, like in the `catalogRelationWarningEntityCard` and `catalogProcessingErrorsEntityCard` default filter functions.

Reply:
Once this other pull request is merged, the filter type for entity contents needs to be updated as well.
https://github.com/backstage/backstage/pull/22752

Reply:
This is not necessary anymore, see: https://github.com/backstage/backstage/pull/22851#discussion_r1486050105